.github/workflows/tests.yaml

```diff
@@ -44,6 +45,11 @@ jobs:
           sudo snap install --classic node
           sudo snap install --classic pyright
           make test-pyright
+      - name: Run linkcheck, woke, spelling
+        run: |
+          sudo snap install woke
```

Comment:
If you move the snap installs to just before doing the apt install and add a `--no-wait` and then do `snap watch --last=install` before the first time we use a snap package it'll make the install time negligible.

Reply:
This can be applied uniformly to all projects if the gain is significant.